test/smoke/generated/purview-administration-rest/review/purview-administration-rest.api.md

```diff
@@ -1256,7 +1262,7 @@ interface PrivateLinkServiceConnectionStateOutput {
 
 declare namespace PurviewAccount {
     export {
-        PurviewAccountClient as PurviewAccount,
+        createClient_2 as PurviewAccount,
```

Comment:
@qiaozha we need to change this so that this gets generated as `createClient_2 as createClient`. instead of PurviewAccount